manifests/00-custom-resource-definition.yaml

```diff
@@ -351,7 +359,7 @@ spec:
                                   type: string
                                 maxLength:
                                   default: 1024
-                                  description: "maxLength is the maximum length of the syslog message \n If this field is empty, the maxLength is set to \"1024\"."
+                                  description: "maxLength is the maximum length of the log message. \n Valid values are integers in the range 480 to 4096, inclusive. \n When omitted, the default value is 1024."
```

Comment:
@dafsjr why is the maximum different for syslog?

Reply:
this file is the same of vendor/github.com/openshift/api/operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml, it is autogenerated/updated, Miciah asked me to modify the message for syslog too in the PR for openshift/api.

Reply:
But don't you want the same numeric maximum for syslog and sidecar log?

Reply:
sorry, I understood a total different thing, I will reply in slack

Reply:
I checked with @Miciah and this should be fine.